acceptance/framework/vault/helpers.go

```diff
@@ -116,9 +121,24 @@ func configureEnterpriseLicenseVaultSecret(t *testing.T, vaultClient *vapi.Clien
 	require.NoError(t, err)
 }
 
-// configureKubernetesAuthRole configures a role for the component for the Kubernetes auth method
+// ConfigureEnterpriseLicenseVaultSecret stores it in vault as a secret and configures a policy to access it.
+func ConfigureSnapshotAgentSecret(t *testing.T, vaultClient *vapi.Client, cfg *config.TestConfig, config []byte) {
+	logger.Log(t, "Creating the Snapshot Agent Config secret")
+	params := map[string]interface{}{
+		"data": map[string]interface{}{
+			"config": config,
+		},
+	}
+	_, err := vaultClient.Logical().Write("consul/data/secret/snapshot-agent-config", params)
+	require.NoError(t, err)
+
+	err = vaultClient.Sys().PutPolicy("snapshot-agent-config", snapshotAgentPolicy)
+	require.NoError(t, err)
+}
+
+// ConfigureKubernetesAuthRole configures a role for the component for the Kubernetes auth method
```

Comment:
for any component? Just wasn't sure how to read the comment! Maybe:
```suggestion
// ConfigureKubernetesAuthRole configures a role for a component for the Kubernetes auth method
```

Reply:
can we add to the comment that this adds a role to vault? When reading the test I was trying to recall what this function does and I think adding that to the comment could help :)